### PR TITLE
Add methods + tests for host muting

### PR DESCRIPTION
**Pull request by wang-arthur**

### CODE REVIEW

File: lib/dogapi/v1/monitor.rb [hunk 1]
```diff
@@ -231,6 +231,33 @@ def get_all_downtimes(options = {})
         end
       end
 
+      #
+      # HOST MUTING
```
Comment (yannmh) [inline lib/dogapi/v1/monitor.rb]: Shall we have a separate `host.rb` file ?

Comment (miketheman) [inline lib/dogapi/v1/monitor.rb]: I think that would make sense if we interact with Hosts as objects - it appears that these methods are added to interact with the `/monitor` api interface.

Comment (wang-arthur) [inline lib/dogapi/v1/monitor.rb]: Indeed; this endpoint is actually a convenience wrapper around downtime scheduling.

**Comment by yannmh** [comment]
Great, thanks @wang-arthur. Looks ready to go to me.

**Pull request merged**